README.md

```diff
@@ -111,3 +170,29 @@ This component will look for atomic indicators of relationships. Atomic indicato
 This component will utilize ML algorithms to train models based on a labeled dataset and then assign every new unknown incoming artefact (depending on the type of artefact) to one of the trained malicious clusters/classes.
 
 #### Final Relationships Generator
+###### Query Example
```

Comment:
I would write this in a way that is useful to a user trying to use your tool. This should be very high level. For example, "This part identifies the overall relationship between objects. We call these "final relationships." These relationships will look at follows
<picture>
We generate the final relationships by examining the primary relationships and computing and overall score. We search xyz number of levels. we use xyz methods

Retreiving relationships
Relationships can be retreived by 
d3.js graph
json

json meaning
the following structure

bla